src/solidClientHelpers/resource.test.js

```diff
@@ -310,14 +311,32 @@ describe("deleteResource", () => {
       deleteResource(resourceInfo, policiesContainer, fetch)
     ).resolves.toBeUndefined();
   });
+});
 
-  it("lets through other errors when deleting the policy resource", async () => {
-    getPolicyUrl.mockReturnValue("https://example.org/examplePolicyUrl");
-    SolidClientFns.deleteFile.mockResolvedValueOnce().mockImplementation(() => {
-      throw new Error("500");
+describe("deletePoliciesContainer", () => {
+  const mockDeleteContainer = jest
+    .spyOn(SolidClientFns, "deleteContainer")
+    .mockImplementation(jest.fn());
+
+  const fetch = jest.fn();
+  const containerIri = "https://example.org/policies/resource/";
+
+  test("it deletes the policies container", async () => {
+    await deletePoliciesContainer(containerIri, fetch);
+
+    expect(mockDeleteContainer).toHaveBeenCalledWith(containerIri, {
+      fetch,
     });
+  });
+
+  it("ignores 403 errors when deleting the policy resource", async () => {
+    SolidClientFns.deleteContainer
+      .mockResolvedValueOnce()
+      .mockImplementation(() => {
```

Comment:
I used this pattern self a lot, realized it's simpler to just write `.mockRejectedValue("403")` instead of using `.mockImplementation`.